### DISCUSSION TITLE
Подробно о точке выдачи (v3.8)

### DISCUSSION THREAD

**Discussion by kaavain** [cdek-it/widget#23] [Ideas]
Совсем в старом виджете к пункту выдачи было описание и даже фото. В новом ничего нет. Это выглядит хорошо с точки зрения дизайна, и понятно зачем сделано - чтобы не возиться с версткой непредсказуемых по размеру блоков.
Но информация очень важная и нужная, может быть ее выводить по клику на кнопку "подробнее" (которую поместить над "выбрать") накрывающим блоком, как сейчас реализована информация о выборе точки при открытом аккордеоне (z-index +1)? Или все же под "выбрать" так как там все равно много пустого места.

**Comment by vermorag** [comment]
Данная доработка не будет выполнена.
На карте отображается функциональность ПВЗ. Внешний вид офиса можно посмотреть на картах или на официальном сайте cdek.ru.
